config/prod/JKG-s3.yaml

```diff
@@ -0,0 +1,36 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: GatesKI-TestStudy-Bucket
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "sysbio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username, required if AllowWriteBucket=true
+  SynapseUserName: 'jgockley'
+  # (Optional) Allow accounts, groups, and users to access bucket.
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
```

Comment:
if you want to give yourself access to this bucket then add your IAM ARN from your Sandbox account. it would be something like this..

```
  GrantAccess:
    - 'arn:aws:iam::325565585839:root' 
    - 'arn:aws:iam::563295687221:user/jake.gockley@sagebase.org'
```